test/assert_exit.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
```

Comment:
I've extracted this from [heroku-apps](https://github.com/heroku/heroku-apps/blob/master/test/assert_exit.js). It seems to be copied over in several plugins.